### DISCUSSION TITLE
`List.graphql.itemQueryName` and `listQueryName` in documentation but are not implemented

### DISCUSSION THREAD

**Discussion by OlleMattsson** [keystonejs/keystone#8930] [Questions] [answered]
In the code accompanying the [GraphQL section on list configuration](https://keystonejs.com/docs/config/lists#graphql)  there is this example:

```
export default config({
  lists: {
    SomeListName: list({
      graphql: {
        description: '...',
        itemQueryName: '...',
        listQueryName: '...',
        ...
```

The props `itemQueryName` and `listQueryName` are not explained in detail but one can guess at their purpose. It however these are not defined in  the `ListGraphQLConfig` type.

In my current project it would be super useful to have these overrides available. I know about the `plural` option but it doesn't allow renaming the "singular" item query would 

So my question: What's the actual situation here? Are these in the works or do we have a simple case of outdated documentation?

**Comment by dcousens** [comment, marked as answer]
Thanks for pointing this out @OlleMattsson, this is a bug in our documentation.
Fixed in https://github.com/keystonejs/keystone/pull/9644

You can now change the `singular` and `plural` in the newest upcoming version of Keystone.
